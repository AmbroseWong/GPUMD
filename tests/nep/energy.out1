-3.32266 -3.12541
-3.321 -3.14185
-3.31923 -3.15725
-3.31706 -3.17165
-3.31478 -3.18506
-3.31221 -3.19751
-3.30939 -3.20902
-3.30635 -3.2196
-3.30314 -3.22929
-3.29963 -3.23809
-3.29585 -3.24603
-3.29188 -3.25313
-3.28766 -3.25941
-3.28332 -3.26489
-3.27863 -3.26958
-3.27393 -3.27351
-3.26886 -3.27669
-3.26372 -3.27914
-3.25826 -3.28087
-3.2527 -3.2819
-3.24672 -3.28226
-3.24078 -3.28195
-3.23462 -3.281
-3.22803 -3.27941
-3.22144 -3.2772
-3.21471 -3.27439
-3.20768 -3.271
-3.20048 -3.26703
-3.19313 -3.2625
-3.18563 -3.25743
-3.1779 -3.25183
-3.17003 -3.24571
-3.16197 -3.23909
-3.15367 -3.23198
-3.14539 -3.22439
-3.13673 -3.21633
-3.12806 -3.20782
-3.1192 -3.19887
-3.11011 -3.18949
-3.10093 -3.1797
-3.09167 -3.1695
