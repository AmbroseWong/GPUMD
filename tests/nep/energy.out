-1.27275 -1.27229
-1.27239 -1.27196
-1.27185 -1.27143
-1.27092 -1.27053
-1.26981 -1.26945
-1.2688 -1.26847
-1.26662 -1.26633
-1.26533 -1.26507
-1.26397 -1.26374
-1.26119 -1.26103
-1.25873 -1.2586
-1.25503 -1.25497
-1.25308 -1.25302
-1.24773 -1.24772
-1.24364 -1.24366
-1.24127 -1.2413
-1.24064 -1.24065
-1.23538 -1.23542
-1.23181 -1.23184
-1.22515 -1.22512
-1.21647 -1.21643
-1.2166 -1.21651
-1.20829 -1.20818
-1.19744 -1.19731
-1.19408 -1.1939
-1.19213 -1.19197
-1.18626 -1.18604
-1.17502 -1.17489
-1.1693 -1.1692
-1.15361 -1.15345
-1.15489 -1.15478
-1.15529 -1.15522
-1.13546 -1.13545
-1.13172 -1.13202
-1.11353 -1.11388
-1.10766 -1.10824
-1.09466 -1.09557
-1.07545 -1.07629
-1.0651 -1.06599
-1.06414 -1.06526
